app/components-react/pages/layout-editor/LayoutEditor.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import cx from 'classnames';
+import cloneDeep from 'lodash/cloneDeep';
+import styles from './LayoutEditor.m.less';
+import { ELayoutElement, LayoutSlot } from 'services/layout';
+import { $t } from 'services/i18n';
+import { Services } from 'components-react/service-provider';
+import { TextInput } from 'components-react/shared/inputs';
+import { useVuex } from 'components-react/hooks';
+import { useLayoutEditor } from './hooks';
+import TopBar from './TopBar';
+import SideBar from './SideBar';
+import AddTabModal from './AddTabModal';
+
+export default function LayoutEditor() {
+  const { LayoutService } = Services;
+
+  const { slottedElements, setBrowserUrl, currentLayout, showModal } = useLayoutEditor();
+
+  useEffect(() => {
+    if (slottedElements[ELayoutElement.Browser]) {
+      setBrowserUrl(slottedElements[ELayoutElement.Browser]?.src || '');
+    }
+  }, []);
```

Comment:
That doesn't look like the right place for calling `setBrowserUrl`.
If you change the component's state after the initial mount that most cases an antipattern because it forces the component to render twice.